src/components/Auth/Login/Login.react.js

```diff
@@ -40,6 +40,7 @@ class Login extends Component {
     serverUrl: PropTypes.string,
     captchaKey: PropTypes.string,
     isCaptchaEnabled: PropTypes.bool,
+    signupErrorMessage: PropTypes.string,
```

Comment:
Just name it `message` so that it is more generic.

Reply:
@akshatnitd Done sir. Please review it :)